Extension/.vscode/launch.json

```diff
@@ -135,9 +62,54 @@
                 "<node_internals>/**"
             ],
             "outFiles": [
-                "${workspaceFolder}/**/out/**/*.js",
+                "${workspaceFolder}/dist/**",
                 "!**/node_modules/**"
             ]
         }
+    ],
+    "inputs": [
+        {
+            "type": "pickString",
+            "id": "pickScenario",
+            "description": "Select which scenario to debug VSCode tests.",
+            "options": [
+                {
+                    "label": "MultirootDeadlockTest   ",
+                    "value": "${workspaceFolder}/test/scenarios/MultirootDeadlockTest/assets/test.code-workspace"
+                },
+                {
+                    "label": "SimpleCppProject   ",
+                    "value": "${workspaceFolder}/test/scenarios/SimpleCppProject/assets/simpleCppProject.code-workspace"
+                },
+                {
+                    "label": "SingleRootProject   ",
+                    "value": "${workspaceFolder}/test/scenarios/SingleRootProject/assets/"
+                }
+            ]
+        },
+        {
+            "type": "pickString",
+            "id": "pickWorkspace",
+            "description": "Select which workspace scenario to debug VSCode.",
+            "default": "-n",
+            "options": [
+                {
+                    "label": "(Debug without Workspace)   ",
+                    "value": "-n"
+                },
+                {
+                    "label": "MultirootDeadlockTest   ",
```

Comment:
Why are there blank spaces in the labels?

Reply:
When it displays, it's really cramped so adding some spaces makes it readable.

Reply:
@fearthecowboy I don't understand how adding 3 spaces at the end makes it readable -- can it be 1 space?

Reply:
I find it a bit easier to read when I have some spaces between the key and value when it shows the dropdown.


<img width="586" alt="image" src="https://github.com/microsoft/vscode-cpptools/assets/133736/db79dc69-a91f-4ed7-aa54-7d3e323e4c1d">